Content.Client/Construction/ConstructionMenuPresenter.cs

```diff
@@ -282,7 +283,7 @@ private void GenerateStepList(ConstructionPrototype prototype, ItemList stepList
                                     switch (subStep)
                                     {
                                         case MaterialConstructionGraphStep materialStep:
-                                            if (!(prototype.Type == ConstructionType.Item)) stepList.AddItem(Loc.GetString("    {0}.{1}.{2}. Add {3}x {4}.", stepNumber, parallelNumber, subStepNumber++, materialStep.Amount, materialStep.Material), icon);
+                                            if (!(prototype.Type == ConstructionType.Item)) stepList.AddItem(Loc.GetString("    {0}.{1}.{2}. Add {3}x {4}.", stepNumber, parallelNumber, subStepNumber++, materialStep.Amount, materialStep.MaterialPrototype.ID), icon);
```

Comment:
same here

Reply:
Done in https://github.com/space-wizards/space-station-14/pull/3387/commits/47f31aee69126d9ca268f2e7872f494e9bbbf3cd